.python-version

```diff
@@ -1 +1 @@
-3.8.16
+3.11.6
```

Comment:
snuba is on python 3.10, will fix soon but we can't go that far. I would also split out this change tbh

Reply:
I've several questions: Why can't we go far? Why does this version needs to be in-sync with Snuba? Sentry repository relies on 3.11 right now.

Reply:
the main issue being we can't block upgrades of an internal library in our own services.  libraries need to support the lowest common denominator internally which is currently 3.10

Reply:
Makes sense. I'll update it in a follow up pull request.